src/module.ts

```diff
@@ -49,6 +48,7 @@ export default defineNuxtModule({
         logger.info(`Using Tailwind CSS from ~/${relative(nuxt.options.srcDir, cssPath)}`)
         nuxt.options.css.splice(injectPosition, 0, cssPath)
       } else {
+        logger.info('Using default Tailwind CSS file from runtime/tailwind.css')
```

Comment:
I hope this isn't too verbose to be an info log (could be debug). Almost any internal of Nuxt is also adding kinda _defaults_.